### ISSUE TITLE
What about Wii Motion Plus ?

### ISSUE THREAD

**Issue by borrougagnou** [abstrakraft/cwiid#33]
Hi ! 
How can I see "Wii Motion Plus" in your interface ?

**Comment by knoxvillesjoker** [comment]
No.  The 2-in-1 models are not compatible with this library.  Certain communications protocols were updated and this library is basically dead as far as updates go.  

The new plus models changed communications methodologies breaking things.
